tests/unit/GithubUtilsTest.js

```diff
@@ -275,7 +275,6 @@ describe('GithubUtils', () => {
         const basePRList = [
             'https://github.com/Expensify/App/pull/2',
             'https://github.com/Expensify/App/pull/3',
-            'https://github.com/Expensify/App/pull/3',
```

Comment:
Been a little while, but I think this duplicate may have been here intentionally to ensure that we don't end up with duplicates in the issue body.